src/pg_cron.c

```diff
@@ -297,12 +327,17 @@ pg_cron_sigterm(SIGNAL_ARGS)
 /*
  * Signal handler for SIGHUP
  *		Set a flag to tell the main loop to reload the cron jobs.
+ *		Set the desired log_min_messages.
  */
 static void
 pg_cron_sighup(SIGNAL_ARGS)
 {
 	CronJobCacheValid = false;
 
+	/* set the desired log_min_messages */
+	ProcessConfigFile(PGC_SIGHUP);
+	SetConfigOption("log_min_messages", cron_error_severity(CronLogMinMessages),
+										PGC_POSTMASTER, PGC_S_OVERRIDE);
```

Comment:
doing a lot of stuff in the signal handler is sketchy. 

Can you move this into the `if (!CronJobCacheValid)` section and maybe rename the variable?

Reply:
Makes sense, PR updated that way.

Reply:
Makes sense, PR updated that way.